src/components/movieCard/movieCard.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { hot } from "react-hot-loader";
+import "./movieCard.scss";
+import PropTypes from "prop-types";
+
+const MovieCard = (props) => (
+    <article className="movieCard">
+        <div className="movieCard__imgWrapper">
+            <img className="movieCard__img" src={props.src}/>
+        </div>
+        <div className="movieCard__main">
+            <a href="#" className="movieCard__title">{props.title}</a>
+            <span className="movieCard__year">{props.year}</span>
```

Comment:
what if year is not passed? 
maybe add some default props

Reply:
Done